### PR TITLE
Add Adyen to dashboard docs

### PR DESCRIPTION
**Pull request by korycins**

### CODE REVIEW

File: docs/dashboard/configuration/plugins.md [hunk 1]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
 
   - Stripe
 
+  - Adyen
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:35]: Could we maybe sort them?

File: docs/dashboard/configuration/plugins.md [hunk 2]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+- Used to extend the Saleor functionalities
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:37]:
```suggestion
- Used to provide certain core features:
```

File: docs/dashboard/configuration/plugins.md [hunk 3]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+  - Invoicing
+
 In all cases of plugin configuration, you need to provide your credentials. Depending on the gate you are configuring, there might be additional setup required.
+
+## Adyen
+
+### Enabling the adyen integration
+
+- Go to Configuration -> Plugins -> Adyen -> Fill up the fields
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:49]:
```suggestion
Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
```

File: docs/dashboard/configuration/plugins.md [hunk 4]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+- `Merchant Account` - your Adyen account as described [here](https://docs.adyen.com/checkout/get-started#step-1-sign-up-for-a-test-account)
+
+- `Supported currencies` - your currency in ISO 4217 like `GBP`
+
+- `Client Key` - The client key is a public key that uniquely identifies a web service. [See Adyen docs.](https://docs.adyen.com/user-management/client-side-authentication#get-your-client-key)
+
+- `Origin Url` - The origin URL of the page where you are rendering the Drop-in. The origin URL of the page where you are rendering the Drop-in. This should not include subdirectories and a trailing slash. For example, if you are rendering the Drop-in on `https://your-company.com/checkout/payment`, specify here: `https://your-company.com`. [See Adyen docs.](https://docs.adyen.com/checkout/drop-in-web)"
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:57]:
```suggestion
- **Origin URL:** The origin URL of the page where you are rendering the Adyen Drop-in. This should not include subdirectories or a trailing slash. For example, if you are rendering the Drop-in on `https://your-company.com/checkout/payment`, put in `https://your-company.com` as the origin. See [Adyen docs](https://docs.adyen.com/checkout/drop-in-web).
```

File: docs/dashboard/configuration/plugins.md [hunk 5]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+### Enabling the adyen integration
+
+- Go to Configuration -> Plugins -> Adyen -> Fill up the fields
+
+- `Merchant Account` - your Adyen account as described [here](https://docs.adyen.com/checkout/get-started#step-1-sign-up-for-a-test-account)
+
+- `Supported currencies` - your currency in ISO 4217 like `GBP`
+
+- `Client Key` - The client key is a public key that uniquely identifies a web service. [See Adyen docs.](https://docs.adyen.com/user-management/client-side-authentication#get-your-client-key)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:55]:
```suggestion
- `Client Key` - The client key is a public key that uniquely identifies a web service. [See Adyen docs](https://docs.adyen.com/user-management/client-side-authentication#get-your-client-key).
```

File: docs/dashboard/configuration/plugins.md [hunk 6]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+## Adyen
+
+### Enabling the adyen integration
+
+- Go to Configuration -> Plugins -> Adyen -> Fill up the fields
+
+- `Merchant Account` - your Adyen account as described [here](https://docs.adyen.com/checkout/get-started#step-1-sign-up-for-a-test-account)
+
+- `Supported currencies` - your currency in ISO 4217 like `GBP`
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:53]:
```suggestion
- **Supported currencies:** your currency as an ISO 4217 3-letter code (eg. `USD`, `EUR`, `GBP`)
```

File: docs/dashboard/configuration/plugins.md [hunk 7]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
 In all cases of plugin configuration, you need to provide your credentials. Depending on the gate you are configuring, there might be additional setup required.
+
+## Adyen
+
+### Enabling the adyen integration
+
+- Go to Configuration -> Plugins -> Adyen -> Fill up the fields
+
+- `Merchant Account` - your Adyen account as described [here](https://docs.adyen.com/checkout/get-started#step-1-sign-up-for-a-test-account)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:51]:
```suggestion
- **Merchant Account:** your Adyen account identifier as described in the [getting started with Adyen guide](https://docs.adyen.com/checkout/get-started#step-1-sign-up-for-a-test-account)
```

File: docs/dashboard/configuration/plugins.md [hunk 8]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+- `Supported currencies` - your currency in ISO 4217 like `GBP`
+
+- `Client Key` - The client key is a public key that uniquely identifies a web service. [See Adyen docs.](https://docs.adyen.com/user-management/client-side-authentication#get-your-client-key)
+
+- `Origin Url` - The origin URL of the page where you are rendering the Drop-in. The origin URL of the page where you are rendering the Drop-in. This should not include subdirectories and a trailing slash. For example, if you are rendering the Drop-in on `https://your-company.com/checkout/payment`, specify here: `https://your-company.com`. [See Adyen docs.](https://docs.adyen.com/checkout/drop-in-web)"
+
+- `Live` - Leave it blank when you want to use test env. [See Adyen docs for production env](https://docs.adyen.com/development-resources/live-endpoints)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:59]:
```suggestion
- **Live:** Leave it blank if you want to use the test environment. See [Adyen docs for production environments](https://docs.adyen.com/development-resources/live-endpoints).
```

File: docs/dashboard/configuration/plugins.md [hunk 9]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+- `Assume all authorizations are automatically captured by Adyen` - All authorized payments will be marked as captured. This should only be enabled if Adyen is configured to auto-capture payments. Saleor doesn’t support the delayed capture Adyen feature.
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:61]:
```suggestion
- **Assume all authorizations are automatically captured by Adyen:** All authorized payments will be marked as captured. This should only be enabled if Adyen is configured to auto-capture payments. Saleor doesn’t support the _delayed capture_ Adyen feature.
```

File: docs/dashboard/configuration/plugins.md [hunk 10]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+- `Automatically capture funds when a payment is made` - If enabled, Saleor will automatically capture funds. If disabled, the funds are blocked but need to be captured manually.
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:63]:
```suggestion
- **Automatically capture funds when a payment is made:** If enabled, Saleor will automatically capture funds. If disabled, the funds are blocked but need to be captured manually.
```

File: docs/dashboard/configuration/plugins.md [hunk 11]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+- `HMAC secret key` - key used to check if a received notification comes from Adyen. The key should be the same as on the Adyen side. Generate new key on the customer-area and copy it to Saleor. [See Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:65]:
```suggestion
- **HMAC secret key:** key used to check if a received notification comes from Adyen. The key should be the same as on the Adyen side. Generate a new key in the Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
```

File: docs/dashboard/configuration/plugins.md [hunk 12]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+- `Notification user` - Additionally you can confirm received notification by checking the provided user in notification. Provide user in Authentication section in your customer area and copy it to Saleor. [See Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:67]:
```suggestion
- **Notification user:** In addition to checking request signatures, you can confirm received notification by checking the provided user in the notification. Provide a user in the _Authentication_ section of your Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
```

File: docs/dashboard/configuration/plugins.md [hunk 13]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+- `Notification password` - Additionally you can confirm received notification by checking the provided user’s password in notification . Provide user’s password in Authentication section in your customer area and copy it to Saleor. [`See Adyen docs`](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:69]:
```suggestion
- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [`Adyen docs`](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
```

File: docs/dashboard/configuration/plugins.md [hunk 14]
```diff
@@ -30,4 +32,43 @@ Saleor arrives with the following plugins already integrated.
...
+
+
+Below you can find the sample configuration of the Adyen.
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:72]:
```suggestion
Below you can find an example configuration of the Adyen plugin:
```

File: docs/dashboard/configuration/plugins.md (hunk 1 above)
Comment (korycins) [reply docs/dashboard/configuration/plugins.md:35]: Yes! 👮

**Pull request closed without merging**